Updated package name

akashdabhi03 opened the pull request.

mathewnik90 commented on app/src/main/config/settings_pedro.plist:
```diff
@@ -118,7 +118,7 @@
 	<!--  -->
 	<key>startup_request_device_owner</key>
-	<string>true</string>
+	<string>false</string>
```
What is this change?

akashdabhi03 replied:
If set to true, the app prompts the user to allow "Modify system settings" permission.

mathewnik90 commented on app/src/main/AndroidManifest.xml:
```diff
@@ -1,36 +1,30 @@
-    <uses-permission android:name="android.permission.RECORD_AUDIO" />
-    <uses-permission android:name="android.permission.CAMERA" />
-    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```
Did we face any issues after removing the permissions?

akashdabhi03 replied:
No we did not face any issues, as of yet, during testing.

mathewnik90 commented on app/src/main/config/settings_diagnostics.plist:
```diff
@@ -118,7 +118,7 @@
 	<!--  -->
 	<key>startup_request_device_owner</key>
-	<string>true</string>
+	<string>false</string>
```
Why are all these parameters changed to false?

akashdabhi03 replied:
Same reason as above.

akashdabhi03 commented:
> Please take a look at the comments given below

I have addressed all the comments. Please review and let me know in case you require additional information.
